README.md

```diff
@@ -46,7 +46,7 @@ For a larger example and features demo:
 
 # Uno Features
 * Supported platforms:
-    * Windows (via the standard UWP Toolkit)
+    * Windows (via the standard [UWP Toolkit](https://docs.microsoft.com/en-us/windows/communitytoolkit/))
```

Comment:
This is not the referenced toolkit, here we're talking about UWP itself, not the Community Toolkit,